detox/e2e/test/messaging/hashtags.e2e.js

```diff
@@ -103,18 +104,30 @@ describe('Hashtags', () => {
         // * Verify tapping on hashtag shows result containing post of hashtag
         await element(by.text(hashtag)).tap();
         await SearchScreen.toBeVisible();
-        await expect(element(by.text(hashtag)).atIndex(1)).toExist();
+        if (isIos()) {
+            await expect(element(by.text(hashtag)).atIndex(1)).toExist();
```

Comment:
Why is this different?

Reply:
I think it has something to do with how the view components are rendered and iOS and Android may be displaying the stack differently. This also happens frequently on alert modals.